tests/integration/sessions_test.go

```diff
@@ -27,5 +27,10 @@ func TestSessions(t *testing.T) {
 		if session == nil {
 			t.Fatalf("Sessions.Read returned nil")
 		}
+
+		token, tokenErr := client.Sessions().CreateTokenFromTemplate(sessionId, "someTemplateSlug")
+		if tokenErr != nil {
+			t.Fatalf("Sessions.CreateTokenFromTemplate returned tokenError: %v", err)
+		}
```

Comment:
IDK if this is going to work as expected given that the token slug may not exist at the clerk account used to run these E2E :sweat_smile:

Reply:
Let's remove the integration test completely. 

You did the right thing to add it here, but your instinct is correct. It won't work unless the template slug exists on the instance.